src/store/Transaction.ts

```diff
@@ -204,56 +269,51 @@ export default {
             const subscriptions: Observable<Transaction[]>[] = [];
             commit('isFetchingTransactions', true);
 
-            if (group == undefined || group === TransactionGroupState.all || group === TransactionGroupState.confirmed) {
-                subscriptions.push(
-                    subscribeTransactions(
-                        TransactionGroupState.confirmed,
-                        transactionRepository.search({
-                            group: TransactionGroup.Confirmed,
-                            address: currentSignerAddress,
-                            pageSize,
-                            pageNumber,
-                            order: Order.Desc,
-                        }),
-                    ),
-                );
-            }
+            subscriptions.push(
+                subscribeTransactions(
+                    TransactionGroupState.confirmed,
+                    transactionRepository.search({
+                        group: TransactionGroup.Confirmed,
+                        address: currentSignerAddress,
+                        pageSize,
+                        pageNumber,
+                        order: Order.Desc,
+                    }),
+                ),
+            );
 
-            // all of the unconfirmed+partial transactions goes in the first page and they are not subject to pagination
-            if (pageNumber === 1) {
-                if (group == undefined || group === TransactionGroupState.all || group === TransactionGroupState.unconfirmed) {
-                    subscriptions.push(
-                        subscribeTransactions(
-                            TransactionGroupState.unconfirmed,
-                            transactionRepository.search({
-                                group: TransactionGroup.Unconfirmed,
-                                address: currentSignerAddress,
-                                pageSize: 100,
-                                pageNumber: 1, // not paginating
-                                order: Order.Desc,
-                            }),
-                        ),
-                    );
-                }
+            subscriptions.push(
```

Comment:
Not sure this is 100% right here. The unconfirmed & partial transaction should always be on top of the tx list table. Especially the partial transaction, which requires the current user to cosign. Could you pls double check see if this is working on the list?

Reply:
Also unconfirmed and partial txs query should be called only if it is the first page. Please notice the `if (pageNumber === 1) ` check above.

Reply:
done